apps/desktop/desktop/src/main/java/bisq/desktop/main/content/chat/message_container/list/message_box/ChatRulesWarningMessageBox.java

```diff
@@ -64,11 +65,11 @@ public ChatRulesWarningMessageBox(
         messageContentVBox.setFillWidth(true);
         messageContentVBox.setAlignment(Pos.CENTER_LEFT);
 
-        closeIcon = new Hyperlink("X");
-        closeIcon.getStyleClass().add("close-icon-link");
+        closeIcon = new BisqMenuItem("close-mini-grey", "close-mini-white");
+        closeIcon.useIconOnly(20);
         closeIcon.setCursor(Cursor.HAND);
         Tooltip.install(closeIcon, closeTooltip);
-        closeIcon.setOnAction(e -> {
+        closeIcon.setOnMouseClicked(e -> {
```

Comment:
NIT: `setOnAction` could be used. Its the more generic handler and preferred if the component support it.